lagrangebench/train/trainer.py

```diff
@@ -253,7 +253,9 @@ def train(
         update_fn = partial(_update, loss_fn=loss_fn, opt_update=self.opt_update)
 
         # init values
-        pos_input_and_target, particle_type = next(iter(loader_train))
+        raw_batch = next(iter(loader_train))
+        raw_batch = jax.tree_map(lambda x: jnp.array(x), raw_batch)  # numpy to jax
+        pos_input_and_target, particle_type = raw_batch
```

Comment:
did this actually have an impact?

Reply:
If the model is the SitL model, then yes. Without converting `particle_type` from a numpy to a jax array, SitL throws an error in the SPH solver call.